km_api/know_me/tests/serializers/test_profile_item_serializer.py

```diff
@@ -27,27 +30,27 @@ def test_create(gallery_item_factory, profile_row_factory, serializer_context):
     assert item.name == data['name']
     assert item.text == data['text']
 
-    assert item.gallery_item == gallery_item
+    assert item.media_resource == media_resource
     assert item.row == row
 
 
-def test_create_other_user_gallery_item(
-        gallery_item_factory,
+def test_create_other_user_media_resource(
+        media_resource_factory,
         profile_factory,
         serializer_context):
     """
-    Users should not be able to attach a gallery item from a different
+    Users should not be able to attach a gallery resource from a different
     profile to their profile item.
     """
-    gallery_item = gallery_item_factory()
+    media_resource = media_resource_factory()
     profile = profile_factory()
 
     serializer_context['profile'] = profile
 
     data = {
-        'gallery_item': gallery_item.pk,
+        'media_resource': media_resource.pk,
         'name': 'My Profile Item',
-        'text': 'I tried to attach a gallery item from another profile.',
+        'text': 'I tried to attach a gallery resource from another profile.',
```

Comment:
Same issue: It doesn't really make sense to reference gallery at all. It is just sample text, so I could be persuaded against the change, but I think it makes sense to change it.